packages/fxa-settings/src/models/contexts/SettingsContext.ts

```diff
@@ -0,0 +1,154 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { gql } from '@apollo/client';
+import React from 'react';
+import config from '../../lib/config';
+import firefox, { FirefoxCommand } from '../../lib/channels/firefox';
+import { createApolloClient } from '../../lib/gql';
+import { Account, GET_PROFILE_INFO } from '../Account';
+import { AlertBarInfo } from '../AlertBarInfo';
+
+export const INITIAL_SETTINGS_QUERY = gql`
```

Comment:
In the admin panel I started moving to the gql to separate .gql.ts files. I feel like this is a bit cleaner and easier to reference when coding in split screen.